src/components/popup/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import styles from './styles.module.scss'
+
+export const Popup = ({
+    sad,
+    happy,
+    exclamation,
+    question,
```

Comment:
looks like 'question' isn't used anywhere, an abandoned variable maybe?

Reply:
Yep, didn't catch it... figured that a question mark wasn't something needed, forgot to clean it up